.clang-format

```diff
@@ -82,7 +82,7 @@ IndentWidth: 8
 InsertBraces: true
 SpaceBeforeParens: ControlStatementsExceptControlMacros
 SortIncludes: Never
-UseTab: Always
+UseTab: ForContinuationAndIndentation
```

Comment:
Do you have an example of how this alters the formatting?   The clang-format documentation is lacking in this regard.

Reply:
Sure, for example the lines https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/cache.h#L77-L78
```c
#define cache_data_enable			arch_dcache_enable
#define cache_data_disable			arch_dcache_disable
```
would become
```c
#define cache_data_enable                       arch_dcache_enable
#define cache_data_disable                      arch_dcache_disable
```

So it won't affect any _indendation_ only alignment